packages/react-server-webpack-plugin/package.json

```diff
@@ -30,7 +30,7 @@
   "devDependencies": {
     "@shopify/react-server": "^0.7.3",
     "@types/webpack-virtual-modules": "^0.1.0",
-    "webpack": "^4.39.1"
+    "webpack": "^4.0.0"
```

Comment:
These both mean the same thing. Is this just a style thing?

Reply:
Or for peer-deps to resolve to at least `4.0.0`

Reply:
Yeah, they are different because version 4.0–4.38 satisfies this but not the old range. In dev dependencies it doesn't matter much but in general having looser version restrictions for library code is desirable